freshlybuiltimagejaano/test_generator.py

```diff
@@ -0,0 +1,66 @@
+from numpy import array,argmax,expand_dims
+from PIL import Image
+from colorama import init,deinit,Fore
+from pickle import load
+from keras.models import load_model
+from keras.applications.xception import Xception
+from keras.preprocessing.sequence import pad_sequences
+from .downloader import model_downloader
+
+class image_description:
```

Comment:
Add docstring to the class and functions.

Reply:
Add Multiline Comment to describe briefly about the class. Adding the name of class in the comment doesn't have any use.